### DISCUSSION TITLE
Fedora package questions

### DISCUSSION THREAD

**Discussion by TH3-S4LM0N** [leftwm/leftwm#982] [Q&A] [answered]
So I was setting up a Fedora (Silverblue) 37 install on my laptop and I read on the readme that the copr is broken atm. So I waited a week or so and it still hasn't changed, so what's actually being done, or more accurately what can i do?
(edit: Looking at the commit history the package was broken in November.)

**Comment by VuiMuich** [comment, marked as answer]
Welcome adventurer!
Sadly there hasn't been any reaction from the package maintainer, despite there have been (to my knowlegde) several attempts to contact them about the issue.
If you have a copr account and willingness to maintain a package, I guess a fork would be appropriate at this point.
